lib/foreman_ansible/engine.rb

```diff
@@ -25,17 +26,33 @@ class Engine < ::Rails::Engine
       Foreman::Plugin.register :foreman_ansible do
         # We need ActiveJob, only available post-1.12 because of Rails 4.2
         requires_foreman '>= 1.12'
-
-        security_block :ansible do
+        security_block :foreman_ansible do
           permission :play_roles,
                      { :hosts => [:play_roles, :multiple_play_roles] },
                      :resource_type => 'Host::Managed'
+          permission :view_ansible_roles,
+                     { :ansible_roles => [:index],
+                       :hosts => [:new, :edit],
```

Comment:
These two permissions (host new/edit and hostgroups new/edit) are overzealous. There is no reason why "view_ansible_roles" should allow users to modify hosts, it's unexpected